_punctuation-grammar/eg-ie-etc-nb.md

```diff
@@ -4,14 +4,22 @@ order: 7
 exclude_from_search: true
 ---
 
-Avoid the Latin abbreviations eg, ie, etc (et cetera) and nb.
+Avoid the Latin abbreviations eg, ie, etc (et cetera) and nb. 
 
 [Don't use italics for Latin]({{ site.baseurl }}/formatting/#italics).
 
-Use 'for example' instead of &#8217;eg&#8217;.
+### eg
 
-If space is limited, for example in a table, you can use &#8217;eg&#8217;.
+Use ‘for example’ instead of ‘eg’. If space is limited, for example in a table, you can use &#8217;eg&#8217;. Only do this if your users can understand it (sometimes screen readers won’t read it correctly).
 
-Only do this if your users can understand it (sometimes screen readers won't read it correctly).
+### ie
 
-Avoid using 'that is' or 'meaning' as an alternative to &#8217;ie&#8217;. Rewrite and restructure your content instead.
+Avoid using ‘that is’ or ‘meaning’ as an alternative to ‘ie’. Rewrite and restructure your content instead.
+
+### etc
+
+Try to restructure the sentence so the user does not need to guess what may be missing. For example, 'We help people to improve their writing skills including grammar, spelling and research.
```

Comment:
Needs closing ' on example, or better still convert both examples to example boxes?